src/web/app/components/adding-question-panel/adding-question-panel.component.html

```diff
@@ -5,10 +5,15 @@
       <button class="dropdown-item" (click)="templateQuestionModalHandler()">Choose from template questions</button>
       <div class="dropdown-divider"></div>
       <ng-container *ngFor="let questionType of FeedbackQuestionType | enumToArray">
-      <button class="dropdown-item" *ngIf="questionType !== FeedbackQuestionType.CONSTSUM"
+        <div *ngIf="questionType !== FeedbackQuestionType.CONSTSUM">
+          <div class="d-flex btn-group justify-content-between" role="group">
+              <button class="dropdown-item" *ngIf="questionType !== FeedbackQuestionType.CONSTSUM"
               (click)="populateAndShowNewQuestionFormHandler(questionType)">
-        {{ questionType | questionTypeName }}
-      </button>
+                    {{ questionType | questionTypeName }}
+            </button>
+            <a type="button" class="btn btn-link" *ngIf="!isLinkDisabled" target="_blank" rel="noopener noreferrer" tmRouterLink="/web/instructor/help" [queryParams]="{questionId: questionType | questionTypeId, section: 'questions'}"><i class="fas fa-info-circle"></i></a>
+          </div>
```

Comment:
Please look into the indentation for the changes in this file and align them with their opening tag. Any nesting of tags should also be one level deeper/ on a new line (i tags)

Reply:
Will do!

Reply:
Reminder to fix the indentation here! Specifically, line 12 seems to have 6 extra spaces. Personally, I think lines 10 and 11 can also be combined into 1 line.

Reply:
@zhaojj2209 just updated the changes!